bin/check-zpool.rb

```diff
@@ -28,23 +28,22 @@ class CheckZPool < Sensu::Plugin::Check::CLI
          default: 7
 
   def run
+    @warnings = []
+    @criticals = []
     zpools = []
     if config[:zpool]
       zpools << SensuPluginsZFS::ZPool.new(config[:zpool])
     else
       zpools = SensuPluginsZFS::ZFS.zpools
     end
-    # Looks odd, but we want to check things that can go critical first
     zpools.each do |zp|
       check_state zp
-    end
-    zpools.each do |zp|
       check_capacity zp
-    end
-    zpools.each do |zp|
       check_vdevs zp
       check_recently_scrubbed zp
     end
+    critical @criticals.first unless @criticals.empty?
```

Comment:
why just the first one? Seems like we should show all criticals.

Reply:
I was trying to keep current behavior, don't mind changing it though

Reply:
Makes sense, in this case since its just output might as well make it better as it has not impact on users needing to change anything.